src/cljs/main/broadfcui/page/method_repo/method/exporter.cljs

```diff
@@ -84,7 +84,7 @@
                  (fn [{:keys [success? parsed-response]}]
                    (if success?
                      (let [configs (map #(assoc % :payload (utils/parse-json-string (:payload %) true)) parsed-response)]
-                       (swap! state assoc :configs configs))
+                       (swap! state assoc :configs configs :preview-config (first (filter #(= (:preview-config-id @state) (config->id+snapshot %)) configs))))
```

Comment:
Reviewer: I could replace `(first (filter...` with a function that returns the first element matching the predicate... didn't see one in a quick look around

Reply:
I think we used to have a `utils/first-matching`, but we deleted it due to not being used. There is a `utils/first-matching-index`. Let's re-add it and use it here.